src/main/resources/spring/camel-context.xml

```diff
@@ -59,6 +59,29 @@
                     <bean ref="initialSavingsEstimationReportService" method="findReportSummaryById(${header.id})" />
                 </route>
             </get>
+            <delete uri="/{id}">
+                <description>Delete a report</description>
+                <route id="report-delete">
+                    <bean ref="analysisService" method="findById(${header.id})"/>
+                    <choice>
+                        <when>
+                            <simple>${body} != null</simple>
+                            <bean ref="analysisService" method="deleteById(${header.id})" />
+                            <setHeader headerName="Exchange.HTTP_RESPONSE_CODE">
+                                <constant>204</constant>
```

Comment:
Could you replace this with [`HttpServletResponse.SC_NO_CONTENT`](https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html#SC_NO_CONTENT)?